### PR TITLE
Translate PULL_REQUEST_TEMPLATE.md to english

### PR DESCRIPTION
**Pull request by s-gbz**
Addresses #107.
We might consider translating the [contribution guide](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md) as well.

### CODE REVIEW

**Comment by s-gbz** [comment]
@silasmahler kindly review this 🥇

**Comment by s-gbz** [comment]
Updated PR to include changes made in #114

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-  - [ ] Code-Snippets wurden mit [Syntax Highlighting](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#syntax-highlighting) versehen
-  - [ ] Der Artikel wurde in der [Vorschau](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#online-preview) geprüft 
-- **Review** (abzuhaken durch den Reviewer)
-  - [ ] Rechtschreibung korrigiert
-  - [ ] Der Artikel wurde in der [Vorschau](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#online-preview) geprüft 
-  - [ ] CCO wurde über neuen Artikel informiert
-  - [ ] Date-Angaben 
-    - [ ] stimmen im Header und im Dateinamen überein
-    - [ ] stehen auf dem heutigen Tag (aber in der Vergangenheit)
+If you make a pull request, please let us know your changes by using the publish checklist below.
```
Comment (florianluediger) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
If you create a pull request, please log your changes by using the publish checklist below. 
```

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
-  - [ ] CCO wurde über neuen Artikel informiert
-  - [ ] Date-Angaben 
-    - [ ] stimmen im Header und im Dateinamen überein
-    - [ ] stehen auf dem heutigen Tag (aber in der Vergangenheit)
+If you make a pull request, please let us know your changes by using the publish checklist below. 
 
+# Publish Checklist
 
+- **Meta** (to be ticked off by the author)
+  - [ ] [Item file stored in the right place](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#dateiname-und-ablageort)
```
Comment (florianluediger) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
  - [ ] [Article text file stored in the correct place](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#dateiname-und-ablageort)
```

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,28 +1,27 @@
...
+  - [ ] [Author info maintained](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#autoren-informationen)
+  - [ ] [Metadata maintained:](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#metadaten)
+    - [ ] "title" assigned
+    - [ ] "author" specified
+    - [ ] "categories" contains at most one of the main categories (Java, Microsoft, Methodology, Architectures, Software Development, Business & People)
```
Comment (florianluediger) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
    - [ ] "categories" contains at most one of the main categories (Java, Microsoft, Methodik, Architekturen, Softwareentwicklung, Business & People)
```

Du kannst die Kategorien hier nicht einfach umbenennen, sonst kommt der Blog damit nicht klar.

File: PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,28 +1,27 @@
...
+    - [ ] "tags" contains (any) matching tags
+- **Text** (to be ticked off by the author)
+  - [ ] [Teaser complies with specifications](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#einleitung--teaser)
+  - [ ] Spelling corrected
+  - [ ] [Indent to one sentence per line](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#ein-satz-pro-zeile)
```
Comment (florianluediger) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
  - [ ] [Each sentence is on its own line](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#ein-satz-pro-zeile)
```

File: PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,28 +1,27 @@
...
+  - [ ] [Images are correctly included in the article](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#bilder)
+  - [ ] [Headings correspond to the specifications](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#%C3%9Cberschriften)
+  - [ ] Code snippets have been provided with [Syntax Highlighting](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#syntax-highlighting)
+  - [ ] The article was reviewed in the [Preview](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#online-preview) 
+- **Review** (to be ticked off by the reviewer)
+  - [ ] Spelling corrected
+  - [ ] The article was reviewed in the [Preview](https://github.com/adessoAG/devblog/blob/master/examples/2017-08-10-blog-post-guide.md#online-preview) 
+  - [ ] CCO was informed about new article
```
Comment (florianluediger) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
  - [ ] CCO was informed about the new article
```

File: PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,28 +1,27 @@
...
+  - [ ] Date specifications 
+    - [ ] match in header and file name
+    - [ ] stand on today (but in the past)
```
Comment (florianluediger) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
    - [ ] are set to today (but in the past)
```

Comment (s-gbz) [reply PULL_REQUEST_TEMPLATE.md]: Done.

**Pull request closed without merging**